config/config.inc.php

```diff
@@ -123,8 +123,7 @@
 define('_THEME_NAME_', $context->shop->theme->getName());
 define('_PARENT_THEME_NAME_', $context->shop->theme->get('parent') ?: '');
 
-define('__PS_BASE_URI__', $context->shop->getBaseURI());
-
+define('__PS_BASE_URI__', htmlentities($context->shop->getBaseURI(), ENT_QUOTES, 'UTF-8'));
```

Comment:
I think this should be escaped when used, not when assigned. What happens if I use this constant on a header redirection? It won't work with html entities.

How about making sure the URI is a valid URI instead?

Reply:
You're right, we need to validate the URI instead change this define :) I will change this.